File: Errors1_1.py
import math

true_val = float(input("\n\nEnter the true value: "))
approx_val = float(input("Enter the approximate value: "))

ea = abs(true_val - approx_val)
er = abs(ea / true_val)
ep = 100 * er

print("\n\nAbsolute Error:", ea)
print("Relative Error:", er)
print("Percentage Error:", ep, "%")

File: Errors1_1.java
import java.util.Scanner;

public class Errors1_1{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("\n\nEnter the true value: ");
        float trueVal = scanner.nextFloat();

        System.out.println("Enter the approximate value: ");
        float approxVal = scanner.nextFloat();

        float ea = Math.abs(trueVal - approxVal);
        float er = Math.abs(ea / trueVal);
        float ep = 100 * er;

        System.out.println("\n\nAbsolute Error: " + ea);
        System.out.println("Relative Error: " + er);
        System.out.println("Percentage Error: " + ep + "%");

        scanner.close();
    }
}
